content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,144 @@
+---
+title:       "Newsletter 103 - Late 2023 news"
+author:      "GeoB99"
+date:        2023-08-06
+tags:        [ newsletter ]
+---
+
+Salutations from the ReactOS project team!
+In the previous newsletter we talked about the overall ReactOS releasing progress scheme and development status of the project, as well as the hiring of our long-term developer Hermès Bélusca-Maïto (HBelusca).
```

Comment:
```suggestion
In the previous posts we talked about the overall ReactOS releasing progress scheme and development status of the project, as well as the hiring of our long-term developer Hermès Bélusca-Maïto (HBelusca).
```
(since the hiring is not part of the previous newsletter)